python/ambassador/fetch/secret.py

```diff
@@ -43,6 +44,8 @@ def _admit(self, obj: KubernetesObject) -> bool:
         return super()._admit(obj)
 
     def _process(self, obj: KubernetesObject) -> None:
+        # self.logger.debug("processing K8s Secret %s", dump_json(dict(obj), pretty=True))
```

Comment:
Why add this log but leave it commented out? Also, why change the log line in `python/ambassador/fetch/fetcher.py` but still leave it commented out?

Reply:
I needed those logs while debugging getting the error text for an invalid Secret to be passed all the way through to the user, which argues for leaving them in – but I don't want to have `dump_json()` called even while debugging is disabled, hence, commenting them out.